live-examples/js-examples/array-unshift.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var array1 = [1, 2, 3];
+
+array1.unshift(4, 5);
+// return new length of the array i.e. 5
```

Comment:
I think it might be worth logging this so we can show the return value ("show, don't tell"):

```
console.log(array1.unshift(4, 5));
// expected output: 5
```

What do you think?